lightning/src/ln/chan_utils.rs

```diff
@@ -255,6 +255,21 @@ pub fn get_htlc_redeemscript(htlc: &HTLCOutputInCommitment, keys: &TxCreationKey
 	get_htlc_redeemscript_with_explicit_keys(htlc, &keys.a_htlc_key, &keys.b_htlc_key, &keys.revocation_key)
 }
 
+/// redeem script for funding transaction output
+pub fn make_funding_redeemscript(our_funding_pubkey: PublicKey, their_funding_pubkey: PublicKey) -> Script {
```

Comment:
Probably best to pass the PubKeys by reference here.

Reply:
done